.gitignore

```diff
@@ -23,3 +23,13 @@ sam.yaml
 
 .env
 .eslintcache
+
+# Generated by Skyfield
+de421.bsp
```

Comment:
```suggestion
# Generated by Playwright
```

That de421.bsp file is a Solar System ephemeris file. It would have been left over from the ACROSS Python code.